recipe/meta.yaml

```diff
@@ -63,6 +63,8 @@ test:
     - gmt-config --all  # [not win]
     - gmt  # [win]
     - gmt grdmath -Rg -I1 0 90 SDIST = dist_to_NP.nc -x2
+    # Test reading/writing a JP2 file (S90E000.earth_relief_05m_p.jp2)
+    - gmt grdcut @earth_relief_05m -R-10/-9/3/5 -G/tmp/reliefcut.jp2=gd:JP2OpenJPEG
```

Comment:
Hmm, this doesn't error with a non-zero exit code at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=978661&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3&l=4457:

```
+ gmt grdcut @earth_relief_05m -R-10/-9/3/5 -G/tmp/reliefcut.jp2=gd:JP2OpenJPEG
grdblend [NOTICE]: Remote data courtesy of GMT data server oceania [http://oceania.generic-mapping-tools.org/]

grdblend [NOTICE]: SRTM15 Earth Relief v2.6 at 05x05 arc minutes reduced by Gaussian Cartesian filtering (26.2 km fullwidth) [Tozer et al., 2019].
grdblend [NOTICE]:   -> Download 180x180 degree grid tile (earth_relief_05m_g): S90W180
ERROR 4: `/home/conda/.gmt/server/earth/earth_relief/earth_relief_05m_g/S90W180.earth_relief_05m_g.jp2' not recognized as being in a supported file format. It could have been recognized by driver JP2OpenJPEG, but plugin gdal_JP2OpenJPEG.so is not available in your installation. You may install it with 'conda install -c conda-forge libgdal-jp2openjpeg'
grdblend [ERROR]: ERROR - Unable to convert SRTM file /home/conda/.gmt/server/earth/earth_relief/earth_relief_05m_g/S90W180.earth_relief_05m_g.jp2 to compressed netCDF format
grdblend [ERROR]: Unable to obtain remote file @S90W180.earth_relief_05m_g.nc
ERROR 4: `/home/conda/.gmt/server/earth/earth_relief/earth_relief_05m_g/S90W180.earth_relief_05m_g.jp2' not recognized as being in a supported file format. It could have been recognized by driver JP2OpenJPEG, but plugin gdal_JP2OpenJPEG.so is not available in your installation. You may install it with 'conda install -c conda-forge libgdal-jp2openjpeg'
grdblend [ERROR]: ERROR - Unable to convert SRTM file /home/conda/.gmt/server/earth/earth_relief/earth_relief_05m_g/S90W180.earth_relief_05m_g.jp2 to compressed netCDF format
grdblend [ERROR]: Cannot find file @S90W180.earth_relief_05m_g.nc
ERROR 4: `/home/conda/.gmt/server/earth/earth_relief/earth_relief_05m_g/S90W180.earth_relief_05m_g.jp2' not recognized as being in a supported file format. It could have been recognized by driver JP2OpenJPEG, but plugin gdal_JP2OpenJPEG.so is not available in your installation. You may install it with 'conda install -c conda-forge libgdal-jp2openjpeg'
grdblend [ERROR]: ERROR - Unable to convert SRTM file /home/conda/.gmt/server/earth/earth_relief/earth_relief_05m_g/S90W180.earth_relief_05m_g.jp2 to compressed netCDF format
grdblend [ERROR]: Cannot find file @S90W180.earth_relief_05m_g.nc
grdblend [ERROR]: File @S90W180.earth_relief_05m_g.nc not found
[Session gmt (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
grdcut [ERROR]: ERROR - Unable to produce blended grid from /tmp/=tiled_333_GX_YV6NSp
[Session gmt (0)]: Error returned from GMT API: GMT_GRID_READ_ERROR (18)
[Session gmt (0)]: Error returned from GMT API: GMT_GRID_READ_ERROR (18)
[Session gmt (0)]: Error returned from GMT API: GMT_GRID_READ_ERROR (18)
+ exit 0
```

@seisman, is there a better to do this JP2 driver check?

Reply:
Maybe just run `gdal-config --formats | grep JP2OpenJPEG`?

Reply:
That won't error with a non-zero exit code either.

Reply:
Ok, this seems to work:

```bash
gmt grdcut @earth_relief_05m -R-10/-9/3/5 -G/tmp/reliefcut.jp2=gd:JP2OpenJPEG
[ -e '/tmp/reliefcut.jp2' ] && exit 0 || exit 1
```

errors as intended at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=978819&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3&l=4480:

```
+ gmt grdcut @earth_relief_05m -R-10/-9/3/5 -G/tmp/reliefcut.jp2=gd:JP2OpenJPEG
grdblend [NOTICE]: Remote data courtesy of GMT data server oceania [http://oceania.generic-mapping-tools.org/]

grdblend [NOTICE]: SRTM15 Earth Relief v2.6 at 05x05 arc minutes reduced by Gaussian Cartesian filtering (26.2 km fullwidth) [Tozer et al., 2019].
grdblend [NOTICE]:   -> Download 180x180 degree grid tile (earth_relief_05m_g): S90W180
ERROR 4: `/home/conda/.gmt/server/earth/earth_relief/earth_relief_05m_g/S90W180.earth_relief_05m_g.jp2' not recognized as being in a supported file format. It could have been recognized by driver JP2OpenJPEG, but plugin gdal_JP2OpenJPEG.so is not available in your installation. You may install it with 'conda install -c conda-forge libgdal-jp2openjpeg'
grdblend [ERROR]: ERROR - Unable to convert SRTM file /home/conda/.gmt/server/earth/earth_relief/earth_relief_05m_g/S90W180.earth_relief_05m_g.jp2 to compressed netCDF format
grdblend [ERROR]: Unable to obtain remote file @S90W180.earth_relief_05m_g.nc
ERROR 4: `/home/conda/.gmt/server/earth/earth_relief/earth_relief_05m_g/S90W180.earth_relief_05m_g.jp2' not recognized as being in a supported file format. It could have been recognized by driver JP2OpenJPEG, but plugin gdal_JP2OpenJPEG.so is not available in your installation. You may install it with 'conda install -c conda-forge libgdal-jp2openjpeg'
grdblend [ERROR]: ERROR - Unable to convert SRTM file /home/conda/.gmt/server/earth/earth_relief/earth_relief_05m_g/S90W180.earth_relief_05m_g.jp2 to compressed netCDF format
grdblend [ERROR]: Cannot find file @S90W180.earth_relief_05m_g.nc
ERROR 4: `/home/conda/.gmt/server/earth/earth_relief/earth_relief_05m_g/S90W180.earth_relief_05m_g.jp2' not recognized as being in a supported file format. It could have been recognized by driver JP2OpenJPEG, but plugin gdal_JP2OpenJPEG.so is not available in your installation. You may install it with 'conda install -c conda-forge libgdal-jp2openjpeg'
grdblend [ERROR]: ERROR - Unable to convert SRTM file /home/conda/.gmt/server/earth/earth_relief/earth_relief_05m_g/S90W180.earth_relief_05m_g.jp2 to compressed netCDF format
grdblend [ERROR]: Cannot find file @S90W180.earth_relief_05m_g.nc
grdblend [ERROR]: File @S90W180.earth_relief_05m_g.nc not found
[Session gmt (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
grdcut [ERROR]: ERROR - Unable to produce blended grid from /tmp/=tiled_333_GX_TH7dZO
[Session gmt (0)]: Error returned from GMT API: GMT_GRID_READ_ERROR (18)
[Session gmt (0)]: Error returned from GMT API: GMT_GRID_READ_ERROR (18)
[Session gmt (0)]: Error returned from GMT API: GMT_GRID_READ_ERROR (18)
+ '[' -e /tmp/reliefcut.jp2 ']'
+ exit 1
```
Gonna wait to see  if this works on Windows though :slightly_smiling_face:

Reply:
> the test fails on Windows as expected, but on the `grdcut` command rather than the `[ -e '/tmp/reliefcut.jp2']` command...

Maybe the `/tmp` directory doesn't exist on Windows?

Reply:
Oh yeah, maybe just write to the current directory. Feel free to push to this branch, it's late for me already so won't get to it until tomorrow.